.github/workflows/nodejs.yml

```diff
@@ -5,6 +5,31 @@ on:
       - master
   pull_request:
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      SITE_ADDRESS: http://localhost
```

Comment:
If you want you can add `QAW_ARTIFACT_PATH` and upload artifacts so it includes video and browser log artifacts.

https://github.com/qawolf/node-example/blob/master/.github/workflows/qawolf.yml#L42
https://github.com/qawolf/node-example/blob/master/.github/workflows/qawolf.yml#L47-L52